openreview/openreview.py

```diff
@@ -17,7 +18,7 @@ def __init__(self, baseurl=None, process_dir='../process/', webfield_dir='../web
             try:
                 self.baseurl = os.environ['OPENREVIEW_BASEURL']
             except KeyError:
-                self.baseurl = 'http://localhost:3000'
+                self.baseurl = 'http://openreview.net'
```

Comment:
this is a significant change, from localhost --> openreview

I'm interested to hear your opinion on this change. It makes sense because most users of openreviewpy will only ever interact with the openreview.net instance. But it means that, when we are testing things, we'll have to make sure that we're using localhost